app/models/user.rb

```diff
@@ -36,6 +40,46 @@ def self.new_with_session(params, session)
     end
   end
 
+  def active_for_authentication?
```

Comment:
I'm not seeing where this is being set to run before every page/controller action - is it baked into devise?

Reply:
As mentioned in a side conversation, it is called by devise.